src/test/gcp/cloudscheduler.spec.ts

```diff
@@ -0,0 +1,104 @@
+import * as _ from "lodash";
```

Comment:
I see here you're importing it as `_` so let's change the other file to import like this as well.